example/jobserver

```diff
@@ -60,40 +56,47 @@ get '/server/:server' => sub {
     }
     else
     {
-      $self->render_not_found;
+      $self->render('not_found');
```

Comment:
This should be `$self->reply->not_found` so that it returns a 404.  See https://metacpan.org/pod/Mojolicious::Plugin::DefaultHelpers#reply->not_found